Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Enrichment.yml

```diff
@@ -885,6 +983,4 @@ outputs:
   type: unknown
 tests:
 - No tests (auto formatted)
-fromversion: 6.8.0
-contentitemexportablefields:
-  contentitemfields: {}
+fromversion: 6.5.0
```

Comment:
```suggestion
fromversion: 6.8.0
```

Reply:
https://app.circleci.com/pipelines/github/demisto/content/245929/workflows/16375176-809b-422a-9ea7-4f133ad00791/jobs/693378?invite=true#step-119-1145
 
this change will fix this

Reply:
Thanks for the suggestions. Should have the new changes in the next commit.

Reply:
Addressed in f3c6bce. Thanks @capanw!